opentelemetry-javaagent/src/main/java/io/opentelemetry/javaagent/OpenTelemetryAgent.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.opentelemetry.auto.bootstrap;
+package io.opentelemetry.javaagent;
```

Comment:
Are you sure we don't want to keep `auto` part? `io.opentelemetry.auto.javaagent`?

Reply:
I did this to be forward compatible with the proposal in https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/754#issuecomment-671011320, in particular this part at the end of that comment:

* io.opentelemetry.javaagent.tooling
* io.opentelemetry.javaagent.bootstrap